test/utils/array.js

```diff
@@ -145,7 +145,7 @@ test('filterBy', (t) => {
   t.deepEqual(out, [obj3], 'Finds items with string truthiness');
 
   out = filterBy(ary, 'bar', false);
-  t.deepEqual(out, [obj2], 'Finds items with falsey values');
+  t.deepEqual(out, [obj2], 'Finds items with falsy values');
```

Comment:
both were used in the repo

Reply:
Either is valid, but falsy is more common from my experience.